package day44_Exceptions;

public class _02_Checked_Exception {
    public static void main(String[] args) {

        System.out.println("Test Started");

       // Thread.sleep(3000);
        System.out.println("Completed");



    }
}
